it/mail/chrome/messenger/accountCreation.properties

```diff
@@ -41,7 +41,7 @@ config_details_found=Sono stati rilevati i tuoi dettagli di configurazione!
 config_unverifiable=Impossibile verificare la configurazione - password o nome utente errati?
 incoming_found_specify_outgoing=Sono stati rilevati i dettagli di configurazione del server in entrata, specifica il nome host d'invio.
 outgoing_found_specify_incoming=Sono stati rilevati i dettagli di configurazione del server in uscita, specifica il nome host di ricezione.
-please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega di inserire i nomi host mancanti.
+please_enter_missing_hostnames=Impossibile ipotizzare le impostazioni - si prega d'inserire i nomi host mancanti.
```

Comment:
CORRECTIONS NEEDED. 

Elision and apocopation should be avoided when unnecessary. Please replace "d'" with "di".